### DISCUSSION TITLE
How do I ask a question or get help on a package

### DISCUSSION THREAD

**Discussion by dssatech** [community/community#147266] [Other Feature Feedback, Questions, & Ideas]
I am trying to use SavvyCAN and can not get the CAN devices to change baud rate, they are stuck at 250K.
Where do I ask a question ?
Where do I get support for this package ?
Where do I get info on how to use this forum ?
no idea what this feedback tool does, but someone please reply to _[removed by moderator]_ (dssatech)
PLEASE REPLY - totally lost newbie.

**Comment by tuves** [comment]
Hi @dssatech,

Thanks for being a part of the GitHub Community.

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [CANopen repo](https://github.com/curvi/CANopen/issues) as @jdevfullstack mentioned in a previous discussion. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
